### PR TITLE
Reawesome/utils fetch: Converted getGroundWaterLevels and getWaterQualityMeasurements...

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

File: R/uvhydrograph-data.R [hunk 1]
```diff
@@ -22,7 +22,11 @@ parsePrimaryUVData <- function(data, month, useDownsampled=FALSE) {
   est_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesName, estimatedOnly=TRUE), month)
   uncorr_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$primarySeriesRawName ), month)
   comp_UV <- subsetByMonth(getTimeSeries(data, timeSeriesNames$comparisonSeriesName ), month)
-  water_qual <- subsetByMonth(getWaterQualityMeasurements(data), month)
+  water_qual <- tryCatch({
+    subsetByMonth(readWaterQualityMeasurements(data), month)
```
Comment (lindsayplatt) [inline R/uvhydrograph-data.R:26]: would you want `subsetByMonth` to occur outside of the tryCatch for `readWaterQualityMeasurements`? I guess it depends on if you want the code to actually stop if there's an error with subsetting or just return an NA data frame.

**Comment by lindsayplatt** [comment]
tryCatch implementation looks correct to me :+1:

File: R/uvhydrograph-data.R (hunk 1 above)
Comment (zmoore-usgs) [reply R/uvhydrograph-data.R:26]: That's a good point, probably best to split this up into 2 separate things.

Comment (thongsav-usgs) [reply R/uvhydrograph-data.R:26]: The one liner is ok if errors from either function would be handled the same way. They only really need to be split if we have different responses to different error conditions.

**Pull request merged**